docs/cookbook/listening_to_core_events.md

```diff
@@ -51,13 +51,13 @@ You now need to register `OnPublishSlot` as a service in the ServiceContainer an
 In `services.yml` (in your bundle):
 
 ``` yaml
-parameters:
-    my.onpublish_slot.class: Acme\ExampleBundle\Slot\OnPublishSlot
- 
 services:
-    my.onpublish_slot:
-        class: '%my.onpublish_slot.class%'
-        arguments: ['@ezpublish.api.service.content']
+    _defaults:
+        autowire: true
+        autoconfigure: false
```

Comment:
@arfaram  Why not `autoconfigure` as well? Although right now for Slots this is not gonna work out of the box (meaning we still need to keep `tags` below), we rather aim to support that in the future.

Reply:
yes @alongosz the SignalSlotPass require the `signal` attribute.

Reply:
Here you can also keep `autoconfigure: true` to be consistent with the other example.
It does NOT indicate that the Slot will be autoconfigured. Autoconfiguration of Slots is not yet supported.
So you still need to keep
```yaml
        tags:
            - { name: ezpublish.api.slot, signal: ContentService\PublishVersionSignal }
```
which will NOT be overridden by any autoconfiguration since no such setup exists.
This is about advocating best practices and while for this example is not necessary it comes together nicely when we see the second example. Having it different in both examples could be confusing for some developers.